modulefiles/module_base.hera.lua

```diff
@@ -46,12 +46,7 @@ load(pathJoin("cdo", "1.9.5"))
 load(pathJoin("R", "3.5.0"))
 
 -- Temporary until official hpc-stack is updated
-prepend_path("MODULEPATH", "/scratch2/NCEPDEV/ensemble/save/Walter.Kolczynski/hpc-stack/modulefiles/stack")
-load(pathJoin("hpc", "1.2.0"))
-load(pathJoin("hpc-intel", "18.0.5.274"))
 load(pathJoin("hpc-miniconda3", "4.6.14"))
 load(pathJoin("ufswm", "1.0.0"))
-load(pathJoin("met", "9.1"))
-load(pathJoin("metplus", "3.1"))
```

Comment:
Actually the verification job ran? 

And there is no met/metplus for the updated intel and I thought I understood from you that this shouldn't hold up the PR? Maybe I'm mistaken?

Reply:
I think verification "runs" but is just skipped for coupled. I think it may just be several cycles before it kicks in.

I definitely don't want to re-break verification right after I fixed it. I'm trying to push hpc-stack to get the PR in to add MET and MET+ to the official installations, but if you have to use the updated compiler and loading the old one at the end unloads the stuff you need, I can try to build MET and MET+ myself. Right now I'm pointing to a hacked version of the system one in a custom stack location.

Reply:
I don't want to break verification either, but it seems ran and loading the old intel is not going to work. Please let me know what the best path forward is.

Reply:
Okay, let me see what I can do about building MET and MET+.